client/src/parser/parser.js

```diff
@@ -0,0 +1,111 @@
+//const {taskLib} = require('./rpaTaskToLibDict.jsx')
```

Comment:
See comment on same file at different location. Please use the comments I commented there and remove one of the files, as I do not think that it is needed twice

Reply:
This parser is obsolete and I removed it.